<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_AssetsManager</class>
 <widget class="QDialog" name="GUI_AssetsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>330</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>660</width>
    <height>330</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Assets Manager</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <property name="batchSize">
    <number>100</number>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="renameButton">
      <property name="text">
       <string>Rename...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeButton">
      <property name="text">
       <string>Remove...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeAllButton">
      <property name="text">
       <string>Remove all...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportDatButton">
      <property name="text">
       <string>Export to DAT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="importDatButton">
      <property name="text">
       <string>Import from DAT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="selectAllButton">
      <property name="text">
       <string>Select all</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="unselectAllButton">
      <property name="text">
       <string>Unselect all</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toggleSelectionButton">
      <property name="text">
       <string>Toggle selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="preview">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
